OrbitGl/Batcher.h

```diff
@@ -105,6 +105,13 @@ class Batcher {
   void AddShadedBox(Vec2 pos, Vec2 size, float z, const Color& color,
                     std::shared_ptr<Pickable> pickable,
                     ShadingDirection shading_direction = ShadingDirection::kLeftToRight);
+  void AddRoundedBox(Vec2 pos, Vec2 size, float z, float radius, const Color& color,
```

Comment:
@reichlfl didn't like to add methods to Batcher who includes several primitives. Only a comment.